src/components/demoComponent.js

```diff
@@ -11,6 +11,7 @@ export default class DemoComponent extends Component {
     this.width = 100
     this.height = 100
     this.gap = 4
+    this.chartType = 'pi'
```

Comment:
please be so kind and follow the naming conventions and rename your chart type to one of the following options

- Scatter Plot
- Line Chart
- Bar Chart
- Pie Chart
- Donut Chart
- Bubble Chart
- Area Chart
- Radar Chart
- Mixed Chart